File: 99乘法表.c
/**
 * Creation         :       2018.11.18 16:30
 * Last Reversion   :       2018.11.18 16:37
 * Author           :       Lingyong Smile {smilelingyong@163.com}
 * File Type        :       c
 * -----------------------------------------------------------------
 * 99 multiplication table
 *      This function is to calculate 99 multiplication table.
 * -----------------------------------------------------------------
 * Crop right @ 2018 Lingyong Smile {smilelingyong@163.com}
 */

#include <stdio.h>

void MultiplicationTable() {
    for (int i = 1; i <= 9; i++) {
        for (int j = 1; j <= i; j++) {
            printf("%d*%d=%2d ", j, i, j * i);
        }
        printf("\n");
    }
}

int main() {
    MultiplicationTable();
    return 0;
}

File: 99乘法表.py
# -*- coding: UTF-8 -*-
# -----------------------------------------------------------------
# Creation         :       2019.03.08 21:26
# Last Reversion   :       2019.03.08 21:26
# Author           :       Lingyong Smile {smilelingyong@163.com}
# File Type        :       c
# -----------------------------------------------------------------
# 99 multiplication table
#      This function is to calculate 99 multiplication table.
# -----------------------------------------------------------------
# Crop right @ 2019 Lingyong Smile {smilelingyong@163.com}


def Multiplication():
    for i in range(1, 10):
        for j in range(1, i + 1):
            print('{}x{}={}\t'.format(j, i, j * i), end='')
        print()


Multiplication()